src/core/heating_cooling_loads.jl

```diff
@@ -973,6 +1003,9 @@ function BuiltInSpaceHeatingLoad(
     if isnothing(annual_mmbtu)
         annual_mmbtu = spaceheating_annual_mmbtu[city][buildingtype]
     end
+    if length(monthly_mmbtu) == 12
+        monthly_mmbtu = convert(Vector{Real}, monthly_mmbtu)
```

Comment:
why do we need this conversion?

Reply:
I found that I did from a test when I input `monthly_mmbtu`. It was not letting me assign Vector{Float64} (type converted from input file) to Vector{Real} which is the type of `monthly_mmbtu`.

Reply:
Testset `Heating and cooling inputs` where I assign `monthly_mmbtu` in line 695.